learningresources/tests/base.py

```diff
@@ -11,14 +11,17 @@
 
 class LoreTestCase(TestCase):
     """Handle often-needed things in tests."""
+    TEST_USERNAME = 'test'
+    TEST_PASSWORD = 'test'
 
     def setUp(self):
         """set up"""
         super(LoreTestCase, self).setUp()
-        self.username = 'test'
-        self.password = 'test'
-        self.user, _ = User.objects.get_or_create(username=self.username)
-        self.user.set_password(self.password)
-        self.user.save()
+        self.user = User.objects.create_user(
+            username=self.TEST_USERNAME, password=self.TEST_PASSWORD
```

Comment:
Okay. They do work as-written. If it's intentional and not likely to bite another developer, then it's fine.

I still have the nit about the redundant `TEST_`, but not enough to not merge. Let me know if you want to change that and I'll merge.